### ISSUE TITLE
agent-cypress will only use the branch-name from cypress.json and not recognize changes to the config

### ISSUE THREAD

**Issue by mwehnert** [Visual-Regression-Tracker/Visual-Regression-Tracker#92]
We have set a default `branch` in `cypress.json` for VRT. 
When we change the cypress config as [documented](https://docs.cypress.io/api/plugins/configuration-api.html#Usage) from the `/cypress/plugins/index.js` file (and afterwards initialize VRT with the updated config), these changes are not recognized. VRT still uses the default branch name.

Here is our plugins index.js:
```js
const {
  addVisualRegressionTrackerPlugin,
} = require('@visual-regression-tracker/agent-cypress/dist/plugin');

module.exports = async (on, config) => {
  // eslint-disable-next-line no-param-reassign
  config.env.visualRegressionTracker.branch = 'test1234';
  addVisualRegressionTrackerPlugin(on, config);
  return config;
};
```

**Comment by pashidlos** [comment]
should be fixed in version `3.0.3`

**Comment by pashidlos** [comment]
thanks for report!
